package.json

```diff
@@ -145,7 +145,7 @@
     "cypress-merge": "cd cypress && mochawesome-merge ./gui-test-screenshots/cypress_report*.json > ./gui-test-screenshots/cypress.json",
     "cypress-postreport": "yarn cypress-merge && yarn cypress-generate",
     "dev": "yarn clean && NODE_ENV=development NODE_OPTIONS=--max-old-space-size=8192 yarn ts-node node_modules/.bin/webpack serve --progress",
-    "export-pos": "./i18n-scripts/export-pos.sh",
+    "export-pos": "./i18n-scripts/export-pos.sh && ./i18n-scripts/fix-po-files.sh",
```

Comment:
This cannot be fixed in export-pos.Sh, instead of adding another script file?

Reply:
Yes